src/main/java/plugins/nate/smp/utils/TellerUtils.java

```diff
@@ -0,0 +1,57 @@
+package plugins.nate.smp.utils;
+
+import org.bukkit.NamespacedKey;
+import org.bukkit.entity.*;
+import org.bukkit.persistence.PersistentDataType;
+import plugins.nate.smp.SMP;
+
+public class TellerUtils {
+    public static final NamespacedKey TELLER_TYPE_KEY = new NamespacedKey(SMP.getPlugin(), "tellerType");
+    public static final NamespacedKey PARENT_TELLER_KEY = new NamespacedKey(SMP.getPlugin(), "parentTeller");
+    public static char CURRENCY_SYMBOL = '\u00D0';
+
+    public static void createDepositTeller(Player player) {
+        player.getWorld().spawn(player.getLocation(), Villager.class, (villager) -> {
+            configureTeller(villager, "DEPOSIT", player);
+            createTextDisplay(player, villager, "Deposit Diamonds (" + CURRENCY_SYMBOL + ")");
+        });
+    }
+
+    public static void createWithdrawTeller(Player player) {
+        player.getWorld().spawn(player.getLocation(), Villager.class, (villager) -> {
+            configureTeller(villager, "WITHDRAW", player);
+            createTextDisplay(player, villager, "Withdraw Diamonds (" + CURRENCY_SYMBOL + ")");
```

Comment:
```suggestion
            createTextDisplay(player, villager, "(" + CURRENCY_SYMBOL + ") Diamond Teller");
```
Since this teller can handle both withdraw and deposit, it should be renamed to a diamond teller.